src/Tasks/Microsoft.NET.Build.Tasks/NETSdkError.cs

```diff
@@ -2,16 +2,20 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.Build.Framework;
-using Newtonsoft.Json;
-using System;
 using System.Globalization;
 
 namespace Microsoft.NET.Build.Tasks
 {
     /// <summary>
     /// Provides a localizable mechanism for logging an error from the SDK targets.
     /// </summary>
-    public class NETSdkError : TaskBase
+    public class
+#if EXTENSIONS
```

Comment:
Is there a reason this class needs a different name for each assembly?  If so, it would be nice to have a comment explaining why this is.